libcloudforensics/providers/gcp/forensics.py

```diff
@@ -114,7 +114,7 @@ def StartAnalysisVm(project,
         which disk type to use to create the disk. Use pd-standard for a
         standard disk and pd-ssd for a SSD disk.
     cpu_cores (int): The number of CPU cores to create the machine with.
-    attach_disk (list(GoogleComputeDisk)): Optional. List of disks to attach.
+    attach_disks (list(str)): Optional. List of disk names to attach.
```

Comment:
```suggestion
    attach_disks (list[str]): Optional. List of disk names to attach.
```

I think that's what the log2timeline convention is.

Reply:
Changed for this PR; opened #128 so we can enforce this across the codebase in a separate PR.